Assets/Scripts/Handler/PauseHandler.cs

```diff
@@ -1,10 +1,12 @@
-using UnityEngine;
-
-public class PauseButton : MonoBehaviour {
-    [SerializeField] private GameObject pauseMenuUI;
-
-    public void PauseButton_OnClick() {
-        PauseController.Instance.GamePaused = !PauseController.Instance.GamePaused;
-        pauseMenuUI.SetActive(PauseController.Instance.GamePaused);
-    }
-}
+using UnityEngine;
+
+public class PauseHandler : MonoBehaviour
```

Comment:
This is more accurate:
`PauseHandler` -> `TogglePauseMenuHandler`
`Pause()` -> `TogglePauseMenu()`

Reply:
Ill do that right now